articles/backup/active-directory-backup-restore.md

```diff
@@ -18,7 +18,12 @@ This article outlines the proper procedures for backing up and restoring Active
 
 - Make sure at least one domain controller is backed up. If you back up more than one domain controller, make sure all the ones holding the [FSMO (Flexible Single Master Operation) roles](/windows-server/identity/ad-ds/plan/planning-operations-master-role-placement) are backed up.
 
-- Back up Active Directory frequently. The backup should never be more than the tombstone lifetime (by default 60 days), because objects older than the tombstone lifetime will be "tombstoned" and no longer considered valid.
+- Back up Active Directory frequently. The backup should never be more than the tombstone lifetime (TSL) because objects older than the tombstone lifetime will be "tombstoned" and no longer considered valid.
```

Comment:
*You may choose to modify it to:*

- Back up Active Directory frequently. The backup age must not be more than the tombstone lifetime (TSL), because objects older than TSL are **tombstoned** and  are not considered valid.

*Comment: Once a name been given in abbreviation, we may continue to use the abbrebiation throughout*